recipes/simpleitk/build.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Can you add `set -e` to this please?

Reply:
That is unnecessary as conda-build sets it.